src/slices/donation/donation.ts

```diff
@@ -60,6 +60,13 @@ const donation = createSlice({
     },
 
     setSplit: (state, { payload }: PayloadAction<number>) => {
+      if (state.recipient?.hide_bg_tip) {
```

Comment:
In reference to https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2775/files#r1519591055, `setSplit` is already responsible for deciding on the next step, so adding the condition here is natural.